### PR TITLE
Update publish-snapshot.yml

### PR DESCRIPTION
**Pull request by ntoll**
## Description

Ensure playwright is installed.

## Changes

To the publish-snapshot.yml action.

## Checklist

-   [ ] All tests pass locally
-   [ ] I have updated `docs/changelog.md`
-   [ ] I have created documentation for this(if applicable)

### CODE REVIEW

File: .github/workflows/publish-snapshot.yml [hunk 1]
```diff
@@ -41,7 +41,7 @@ jobs:
                       ${{ runner.os }}-
 
             - name: Install Dependencies
-              run: npm install
+              run: make setup
 
             - name: Build Pyscript.core
               run: npm run build
```
Comment (WebReflection) [inline .github/workflows/publish-snapshot.yml:47]: I think here you need `npx playwright install` before `npm run build` here

File: .github/workflows/publish-snapshot.yml [hunk 2]
```diff
@@ -41,7 +41,7 @@ jobs:
                       ${{ runner.os }}-
 
             - name: Install Dependencies
-              run: npm install
+              run: make setup
```
Comment (WebReflection) [inline .github/workflows/publish-snapshot.yml:44]: no, the `npm install` was OK, you are just missing the `npx playwright install` here right after

**Comment by WebReflection** [comment]
@ntoll we have **two** `playwright install` needed, one is for the top level folder, one is for the `pyscript.core` one ... these two integration tests are different because the one we use on the top level folder only deal with *pyodide*, the one in `pyscript.core` folder instead ensure *MicroPython* also work, but we haven't instrumented our test suite (yet?) to deal with it.

Playwright doesn't understand two installs in two different folders, each folder needs its own, and none of those installs can be skipped or should be removed.

Current GitHub Action in Pyscript project has been fine for days, so that's the only difference since last release candidate.

File: .github/workflows/publish-snapshot.yml (hunk 1 above)
Comment (ntoll) [reply .github/workflows/publish-snapshot.yml:47]: I've put it as part of the "install deps" step (feels like the right place to be).

**Review by WebReflection — APPROVED** [review]

File: .github/workflows/publish-snapshot.yml (hunk 1 above)
Comment (WebReflection) [reply .github/workflows/publish-snapshot.yml:47]: it's OK, that should work too

**Pull request merged**